packages/eas-cli/src/commands/branch/publish.ts

```diff
@@ -171,11 +212,10 @@ export default class BranchPublish extends Command {
         })
       );
     }
-    assert(name, 'branch name must be specified.');
-
-    const { id: branchId, updates } = await viewUpdateBranchAsync({
+    assert(branchName, 'branch name must be specified.');
```

Comment:
I think we could move this `assert` inside the conditional above it. It's a check to make sure that the user chose a branch via an interactive prompt, so it seems like it belongs in there, since the other flow will get the git branch or create a random branch.

Reply:
It's also meant as a defensive check in case someone changes the code where it's set automatically.

Seems costless to have it serve both purposes.

Reply:
Well, maybe you are right.